octopus-agent/src/main/java/octopus/teamcity/agent/OctopusBuildInformationBuildProcess.java

```diff
@@ -89,8 +89,7 @@ protected OctopusCommandBuilder createCommand() {
               sharedConfigParameters.get("build.vcs.number"),
               restfulBuild.getBranch().getName(),
               createJsonCommitHistory(restfulBuild),
-              teamCityServerUrl,
-              buildIdString,
+              sharedConfigParameters.get("externalBuildURL"),
```

Comment:
nit: externalBulidURL or externalBuildUrl?

Reply:
changed to make it consistent with the variable naming